### ISSUE TITLE
testplan network creation spec is incomplete

### ISSUE THREAD

**Issue by nonsense** [ipfs/testground#266]
#### Version information:
commit 34be4980e06cda7fa5133dafddbee428694bf6d7

#### Description:
When I try to run more than 300 instances testplan on Docker Swarm, some of the tasks in the service fail with:

```
udsabcqfv02p        tg-dht-find-peers-9cb29ae0-206c-45fd-b089-14ceb18f6f24.329   nonsens3/docker-ping:latest   3.121.222.241       Shutdown            Rejected 23 seconds ago          "            "Err": "node is missing network attachments, ip addresses may be exhausted"
```

I have hardcoded a `ping` container that is present on all workers, just so that I avoid the `docker pull` delay.

We should probably modify the network spec we create:

```
	networkSpec := types.NetworkCreate{
		Driver:         "overlay",
		CheckDuplicate: true,
		// EnableIPv6:     true, // TODO(steb): this breaks.
		Internal:   true,
		Attachable: true,
		Scope:      "swarm",
		Labels: map[string]string{
			"testground.plan":     input.TestPlan.Name,
			"testground.testcase": testcase.Name,
			"testground.runid":    input.RunID,
			"testground.name":     "default", // default name. TODO: allow multiple networks.
		},
	}
```

**Comment by nonsense** [comment]
indeed the network per testplan that we current create is:

```
        "Scope": "swarm",
        "Driver": "overlay",
        "EnableIPv6": false,
        "IPAM": {
            "Driver": "default",
            "Options": null,
            "Config": [
                {
                    "Subnet": "10.0.11.0/24",
                    "Gateway": "10.0.11.1"
                }
            ]
        },
```

**Comment by nonsense** [comment]
This should have been addressed by #268
